sdks/go/pkg/beam/core/typex/fulltype.go

```diff
@@ -96,6 +94,8 @@ func printShortComposite(t reflect.Type) string {
 // New constructs a new full type with the given elements. It panics
 // if not valid.
 func New(t reflect.Type, components ...FullType) FullType {
+	checkTypesNotNil(components)
```

Comment:
<!--thread_id:cc_166105875_t; commit:977a4f133a10d957b90566426581193d93041321; resolved:1-->
<!--section:context-quote-->
> **wcn3** wrote:
> Similar to my previous comments, please rename as 'checkTypes'

<!--section:body-->
See my response to that comment.